<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_select">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="vehicle_type_tab">
     <attribute name="title">
      <string>Vehicle Type</string>
     </attribute>
     <attribute name="toolTip">
      <string>Vehicle Type</string>
     </attribute>
     <widget class="QComboBox" name="vehicle_type_select">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>71</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Select the Vehicle Type</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <item>
       <property name="text">
        <string>LPW LRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MPW LRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HPW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LPW HRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MPW HRL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Truck</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="engine_tab">
     <attribute name="title">
      <string>Engine</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>160</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="current_gdi_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>current_GDI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="current_tds_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>current_TDS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="current_atk_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>current_ATK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="future_atk_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>future_ATK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="future_atk_deac_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>future_ATK_deac</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="future_tds_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>future_TDS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="transmission_tab">
     <attribute name="title">
      <string>Transmission</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="current_6at_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>current_6AT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="current_8at_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>current_8AT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="future_6at_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>future_6AT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="future_8at_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>future_8AT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="electrification_tab">
     <attribute name="title">
      <string>Electrification</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>160</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="mhev_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MHEV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="p2hev_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>P2HEV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pshev_select">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PSHEV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="road_load_tab">
     <attribute name="title">
      <string>Road Load</string>
     </attribute>
     <widget class="QLabel" name="mass_reduction_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass Reduction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="mass_reduction_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="mass_reduction_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="mass_reduction_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mass_reduction_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="mass_reduction_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mass_reduction_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="mass_reduction_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="rolling_reduction_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="rolling_reduction_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="rolling_reduction_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rolling_reduction_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="rolling_reduction_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="rolling_reduction_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="rolling_reduction_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>105</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="aero_reduction_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="aero_reduction_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="aero_reduction_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="aero_reduction_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="aero_reduction_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aero_reduction_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="aero_reduction_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>114</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="engine_sizing_form_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="engine_sizing_max_label">
         <property name="text">
          <string>Max %</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="engine_sizing_max_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="engine_sizing_min_label">
         <property name="text">
          <string>Min %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="engine_sizing_min_select">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="engine_sizing_step_label">
         <property name="text">
          <string>Step %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="engine_sizing_step_select">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="rolling_reduction_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rolling Reduction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="aero_reduction_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Aero Reduction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="engine_sizing_label">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Engine Sizing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_mass_reduction_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>212</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Mass Reduction</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_of_iterations">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_rolling_reduction_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Rolling Reduction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_engine_sizing_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Engine Sizing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="validate_aero_reduction_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Validate Aero Reduction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calculate_iterations_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate # of Iterations</string>
      </property>
     </widget>
     <widget class="QLabel" name="number_of_iterations_label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string># of Iterations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="calculation_time_label">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculation Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="calculation_time">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EPA ALPHA Model REVS3 Command File Generator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/alpha_logo.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/epa_logo.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEngines">
   <property name="text">
    <string>Engines</string>
   </property>
  </action>
  <action name="actionTransmissions">
   <property name="text">
    <string>Transmissions</string>
   </property>
  </action>
  <action name="actionRoad_Loads">
   <property name="text">
    <string>Road Loads</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mass_reduction_max_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_mass_reduction_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass_reduction_min_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_mass_reduction_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rolling_reduction_max_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_rolling_reduction_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rolling_reduction_min_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_rolling_reduction_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aero_reduction_max_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_aero_reduction_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aero_reduction_min_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_aero_reduction_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>engine_sizing_max_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_engine_sizing_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>engine_sizing_min_select</sender>
   <signal>valueChanged(int)</signal>
   <receiver>validate_engine_sizing_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass_reduction_step_select</sender>
   <signal>activated(int)</signal>
   <receiver>calculate_iterations_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rolling_reduction_step_select</sender>
   <signal>activated(int)</signal>
   <receiver>calculate_iterations_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aero_reduction_step_select</sender>
   <signal>activated(int)</signal>
   <receiver>calculate_iterations_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>engine_sizing_step_select</sender>
   <signal>activated(int)</signal>
   <receiver>calculate_iterations_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
